js/scripts.js

```diff
@@ -0,0 +1,49 @@
+var mobileMenuExpand = document.querySelector('#mobilemenuexpand');
+var expanded = document.querySelector('#expanded');
+var sub = document.querySelector('#sub');
+var mobileMenu = document.querySelector('#mobilemenu');
+
+mobileMenu.addEventListener ("click", function(){
+	mobileMenuExpand.style.display = "flex"
+});
+
+var exitMobileMenu = document.querySelector('#exit');
```

Comment:
mmm, that sounds like a good intuition though. is there no way to close the menu after u've opened it? there should be, if not i say bring bax the X